helm-charts/prometheus_values.yaml

```diff
@@ -1,5 +1,14 @@
+serviceAccounts:
+  alertmanager:
+    create: false
+
 server:
   ingress:
     enabled: true
     annotations:
       kubernetes.io/ingress.class: private-nginx
+    hosts:
+      - kofw9hx1qbrq9dj8nuuwiyug6h.prometheus.internal.test.cloud.mattermost.com
```

Comment:
Should this be hard-coded for every k8s cluster?

Reply:
Nope this can be removed. Its automatically added by the Provisioner.

Reply:
forgot
will update :(
thanks for the catch

Reply:
done @gabrieljackson @stylianosrigas